frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverview.tsx

```diff
@@ -132,7 +133,11 @@ const MonitoringOverview: React.FC<MonitoringOverviewProps> = (props) => {
               <>
                 <div className="odc-monitoring-overview__view-monitoring-dashboard">
                   <Link
-                    to={`/dev-monitoring/ns/${resource?.metadata?.namespace}/?workloadName=${resource?.metadata?.name}&workloadType=${resource?.kind}`}
+                    to={
+                      currentPath.startsWith('/k8s')
```

Comment:
I think it would be better if we can do it based on the active perspective. WDYT?

Reply:
@sahil143 The issue with checking last active perspective is that it returns 'admin' on `incognito` while You are on dev

Reply:
@abhinandan13jan Why would last active perspective return `admin` on `incognito` while you're on `dev`? Both localStorage and redux store gets initialized as expected on `incognito` as well otherwise the application wouldn't work.

Reply:
I don't think it's a good idea to check for a specific URL pattern. What if this component is added to some other page later which doesn't conform to the above URL pattern? The whole logic would break then. I think it's better to use `activePerspective` from redux here..

Reply:
I agree with @rohitkrai03. We shouldn't be checking based on the URL pattern. If it is returning `admin` on incognito while in `dev` then it's a bug and we should look to fix it.

Reply:
@sahil143 @rohitkrai03 The issue is, if you copy a dev-perspective URL and open it in incognito, the activePerspective in the redux is set as `admin` . Have a look at the below screenshot
![Screenshot from 2020-08-18 22-49-32](https://user-images.githubusercontent.com/22490998/90544758-3ac40600-e1a5-11ea-9726-7df8d812e7ea.png)

Reply:
@debsmita1 That happens for any devconsole url in incognito.

Reply:
I have updated to use getActivePerspective for convention as well and it seems to work fine